app/src/Controller/ArticlesController.php

```diff
@@ -18,6 +19,7 @@ class ArticlesController extends AppController
      */
     public function index()
     {
+        $this->Authorization->skipAuthorization();
```

Comment:
About skipAuthorization. I think we can set in AppController::beforeFilter()

Therefore, we'll skip the authorization for all controller's action extending the parent AppController.

Refer https://book.cakephp.org/4/en/controllers.html#Cake\Controller\Controller::beforeFilter

It's not necessary to called the skipAuthorization in each controller's action manually. Please check.

Reply:
For action add, edit & delete in ArticlesController, we called the `AuthorizationComponent::authorize($article)`. Therefore, the authorization service will judge based on the ArticlePolicy even the parent class method beforeFilter called the skipAuthorization().